packages/types/src/base.ts

```diff
@@ -22,14 +22,16 @@ export enum ContractTypes {
 export enum ChainTypes {
   Ethereum = 'ethereum',
   Bitcoin = 'bitcoin',
-  Cosmos = 'cosmos'
+  Cosmos = 'cosmos',
+  Osmosis = 'osmosis'
 }
 
 export enum NetworkTypes {
   MAINNET = 'MAINNET',
   TESTNET = 'TESTNET', // BTC, LTC, etc...
   ETH_ROPSTEN = 'ETH_ROPSTEN',
   ETH_RINKEBY = 'ETH_RINKEBY',
+  COSMOS_COSMOSHUB_4 = 'COSMOS_COSMOSHUB_4',
```

Comment:
This isn't used anywhere and refers to the same network type as `COSMOSHUB_MAINNET`.
Do we actually need this?

Reply:
Additional conversation surrounding this a bit for ref: https://github.com/shapeshift/lib/pull/398#discussion_r814946990